docs/concepts/copilot/copilot.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Data Copilot
+id: copilot
+description: The AI assistant for data Pipelines and models
+sidebar_position: 8
+tags:
+  - concepts
+  - copilot
+  - generativeai
+---
+
+Prophecy’s end-to-end platform makes data Pipeline development faster and accessible to more users. On top of that, you have an AI assistant to boost that productivity even further.
+
+## Building Models and Pipelines with Prophecy’s Data Copilot
+
+<div class="wistia_responsive_padding" style={{padding:'62.5% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/t7m3boc575?seo=false?videoFoam=true" title="Design a Pipeline Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+To learn more about what Data Copilot can do, see [Data Copilot AI capabilities](/concepts/copilot/copilot-ai-capabilities).
+
+## Knowledge Graph
+
+Data Copilot works by enhancing the user’s prompt using a knowledge graph.
+
+![Knowledge Graph](img/copilot_knowledge_graph.png)
+
+Our knowledge graph includes metadata for a Project’s entities (such as Datasets, Schemas, Seeds, Models, and Pipelines) and the statistical usages of these entities. We built the knowledge graph intentionally to include metadata but not data. The Dataset structure is included in the knowledge graph but individual Dataset records are not.
+
+![Architecture](img/copilot_arch.png)
+
+Prophecy sends the enhanced Prompt to OpenAI. The model returns SQL or Spark code, and Prophecy verifies whether this code is valid. Once Prophecy fixes the code, Prophecy generates a visual Pipeline and displays this Pipeline to you for review.
```

Comment:
This is the first time in our docs where we are using the word "model" in more than one way. E.g. here "the model" refers to the LLM or GenAI model.  Other uses of "model" on this page are "[SQL models](https://docs.getdbt.com/docs/build/sql-models)"

Reply:
Done.

I added large language model to those instances.